.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
This is a IDEA specific directory and should be ignored on a global level instead of a project level.

Reply:
This whole new file is not needed and should not added to the repo - in my opinion as this a IDE specific file.